webflow-hacks/form-hacks/44/44.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+// when DOM is ready
+document.addEventListener('DOMContentLoaded', () => {
+  const FORM_SELECTOR = '[fs-hacks-element="form"]';
+  const SELECT_SELECTOR = '[fs-hacks-element="select"]';
+  const TOTAL_SELECTOR = '[fs-hacks-element="total"]';
+  const HIDDEN_TOTAL_SELECTOR = '[fs-hacks-element="hidden-total"]';
+  const form = document.querySelector(FORM_SELECTOR);
+  const totalDiv = document.querySelector(TOTAL_SELECTOR);
+  const hiddenInput = document.querySelector(HIDDEN_TOTAL_SELECTOR);
+  if (!form || !totalDiv || !hiddenInput) return;
+  // get all selects
+  const selects = form.querySelectorAll(SELECT_SELECTOR);
+  // listener for form
+  form.addEventListener('input', function (event) {
```

Comment:
I'm curious why you've listened for `input` events at the form level in this solution, but on other solutions like the radios or checkboxes calculators you've listened for events at the fields level.

Reply:
Its easier for the user to extend this solution by adding an arbitrary number of `select` input elements. Basically, he wouldn't need to modify the code if he wants to add more than two selectors. For the listener at form element method, I would need to track subtotals for every selector with a different variable. Hence more complex code?